Cargo.toml

```diff
@@ -26,14 +26,14 @@ version = "0.1.6"
 rust-version = "1.70"
 
 [workspace.dependencies]
-dioxus = { version = "0.4" }
-dioxus-hooks = { version = "0.4" }
-dioxus-html = { version = "0.4" }
-dioxus-router = { version = "0.4" }
-dioxus-desktop = { version = "0.4", features = ["transparent"] }
+dioxus = { git = "https://github.com/DioxusLabs/dioxus" }
```

Comment:
I would suggest using the specific commit from their git here (and below). Even though we pin the crates, a `cargo update`, which is done at times can easily break everything since it would pull from the latest git head